src/entity/units/archer.ts

```diff
@@ -17,14 +17,14 @@ const unit: UnitSource = {
     subtype: UnitSubType.RANGED_LOS,
   },
   unitProps: {
-    damage: 10,
-    attackRange: 500,
-    healthMax: 40,
+    damage: 20,
+    attackRange: 400,
+    healthMax: 20,
     manaMax: 0,
   },
   spawnParams: {
-    probability: 50,
-    budgetCost: 2,
+    probability: 10,
+    budgetCost: 3,
```

Comment:
I could be talked into this.   Double damage is a lot, but I guess golems have a pretty far stamina range so maybe this makes sense? I'd like to hear your reasoning

Reply:
That's a significantly less probability

Reply:
Archers deal practically 0 damage. In my last test with Luna we both felt we could just ignore them entirely, and it became a chore to chew through them since they weren't threats to us.

I think it also makes sense for ranged guys to be squishy and golems to be tankier etc.

Other Archers are WAYYY scarier, despite similar budget costs.

And yes I noticed that about the probability. This makes it more likely to choose interesting units in later levels, rather than mass default-archer